modal/_container_io_manager.py

```diff
@@ -398,7 +398,7 @@ async def run_inputs_outputs(self, input_concurrency: int = 1) -> AsyncIterator[
         async for input_id, function_call_id, input_pb in self._generate_inputs():
             args, kwargs = self.deserialize(input_pb.args) if input_pb.args else ((), {})
             self.current_input_id, self.current_input_started_at = (input_id, time.time())
-            yield input_id, function_call_id, args, kwargs
+            yield input_id, function_call_id, args, kwargs, input_pb.method_name
```

Comment:
1 - fixed!
2 - I can see your point, but it would feel more natural to send it along if the deserialization happened in the receiver (since we send args and kwargs), but then we are basically duplicating this logic in both the async and sync versions so I'm not sure... Can take a look at refactoring after this patch